aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -19,6 +19,8 @@ update_sequence:
         - resource_groups
         - multisig_accounts
         - delegation_pools
+        - cryptography_algebra_natives
```

Comment:
This file is getting deleted by @runtian-zhou but whoever makes it second will resolve the conflicts :)